container/container.go

```diff
@@ -239,6 +241,30 @@ func EnvForTask(taskInfo *mesos.TaskInfo) []string {
 		envVars = append(envVars, "MESOS_HOSTNAME="+getHostname(taskInfo))
 	}
 
+	// We expose the value in the ServiceName and EnvironmentName
+	// labels as an env vars to the container as well.
+	if labels == nil {
+		log.Warn("No ServiceName or EnvironmentName set in Docker labels!")
+		return envVars
+	}
```

Comment:
hey - why is the behaviour different for if there are no labels v if there is no servicename/environmentname labels? first returns none, the other returns stuff from the container or "undefined-servicename"/"dev"